Sources/Commands/SwiftBuildTool.swift

```diff
@@ -173,13 +173,13 @@ public struct SwiftBuildTool: SwiftTool {
         print("USAGE: swift build [mode] [options]")
         print("")
         print("MODES:")
-        print("  -c, --configuration <value>   Build with configuration (debug|release)")
-        print("  --clean <mode>                Delete artifacts (build|dist)")
+        print("  -c, --configuration <value>   Build with configuration (debug|release) [default: debug]")
+        print("  --clean <mode>                Delete artifacts (build|dist) [default: build]")
```

Comment:
obviously that would require changing the code and should be a sep. PR (if at all)
IMO this PR is ok